iconCompiler/config.js

```diff
@@ -1,10 +1,22 @@
 const path = require('path')
-const currentDir = process.cwd()
+const destPath = strPath => path.relative(process.cwd(), strPath)
 
 module.exports = {
-  iconsOutputPath: {
-    svg: path.join(currentDir, 'packages/icons/svg/'),
-    react: path.join(currentDir, 'packages/icons/react/'),
-    vue: path.join(currentDir, 'packages/icons/vue/'),
+  sourcePaths: {
+    svg: destPath('src/icons/SVG'),
+    svgColor: destPath('src/icons/COLORS/SVG'),
+    svgColorToGatsby: destPath('src/icons/COLORS/SVG'),
+    pdf: destPath('src/icons/PDF'),
+    pdfColor: destPath('src/icons/COLORS/PDF'),
+  },
```

Comment:
Any reason to use full caps for folder names?

Reply:
it's the directory structure given by the designers.

Reply:
Copy that 👍